src/index.js

```diff
@@ -97,3 +116,8 @@ module.exports = function lintStaged(logger = console, configPath, debugMode) {
       `)
     })
 }
+
+exports.runAll = runAll
```

Comment:
AFAIK we use `module.export` pattern in this project. Please use it here as well to keep things consistent.